e2e/portalicious/pages/RegistrationsPage.ts

```diff
@@ -0,0 +1,81 @@
+import { expect, Locator } from '@playwright/test';
+import { Page } from 'playwright';
+
+import BasePage from './BasePage';
+import TableComponent from './TableComponent';
+
+class RegistrationsPage extends BasePage {
+  readonly page: Page;
+  readonly tableRows: Locator;
+  readonly selectAllRegistrationsCheckbox: Locator;
```

Comment:
I think these two should be removed, now that we have a TableComponent.

Also, I think the tableComponent should be a member of this page. Something like this:

```suggestion
  readonly table: TableComponent;
```